src/ProgramBuilder.ts

```diff
@@ -176,8 +185,7 @@ export class ProgramBuilder {
         let debouncedRunOnce = debounce(async () => {
             this.logger.log('File change detected. Starting incremental compilation...');
             await this.runOnce();
-            let errorCount = this.getDiagnostics().length;
-            this.logger.log(`Found ${errorCount} errors. Watching for file changes.`);
+            this.logger.log(`Watching for file changes.`);
```

Comment:
The error count will be reported in the `runOnce` method so this was a duplicated message.